lib/daimon_skycrawlers/cli.rb

```diff
@@ -1,8 +1,11 @@
 require 'thor'
+require 'daimon_skycrawlers/generator/new'
 require 'daimon_skycrawlers/version'
 
 module DaimonSkycrawlers
   class CLI < Thor
+    register(Generator::New, "generate", "generate NAME", "Generate new project")
```

Comment:
I want to use `()` here.
Because I want to break this line when this line becomes longer.